apps/portal/components/pagesComponents/_singleImageScreen/SingleImagePageManageAreaTemplate.tsx

```diff
@@ -634,60 +632,52 @@ const SingleImagePageManageAreaTemplate: React.FC<Props> = ({
                         </div>
                       </AppButton>
                     ) : (
-                      <AppButton
-                        className={styles.appButton}
-                        twPadding="tw-px-0 tw-py-3"
-                        twRounded="tw-rounded-none"
+                      <button
```

Comment:
@Anishali2 is there any specific reason why we replaced AppButton with "button" in this line, but keep it AppButton in other places? Why we replaced it / touched it?